README.md

```diff
@@ -74,6 +74,9 @@
       <td>
         <a href="/docs/README.se.md">Svenska</a>
       </td>
+      <td>
+        <a href="/docs/README.pl.md">Polski</a>
+      </td>
       <td align="center">
         <a href="https://github.com/carbon-app/carbon/new/main/docs?filename=docs/README.%3Cnew%20language%3E.md">Add +</a>
       </td>
```

Comment:
@mbiesiad please remove the last empty `<td></td>` to make the table rows line up:
<img width="641" alt="Screenshot 2020-07-23 20 34 15" src="https://user-images.githubusercontent.com/8397708/88358461-e94e6580-cd23-11ea-9b96-208bb61fdee2.png">

Reply:
Good point, thanks!